pkg/cableengine/healthchecker/healthchecker.go

```diff
@@ -26,17 +26,11 @@ import (
 	"github.com/submariner-io/admiral/pkg/log"
```

Comment:
I wonder if we should also include current GW pinger under routeagent pinger,
I mean we can implement for each node (regardless of its GW or not):
 - healthcheck pinger to remote endpoints
 - store status under routeAgentResource CR
 
 The GW component can retrieve pinger status from the corresponding routeAgentResource CR

Reply:
Should we do it now as the g/w health checker is used by other components too, I did not want to change that.

Reply:
I assume we can do it in a follow-up PR
BTW, do you plan to use same ping interval for both GW and non-GW nodes ?

Reply:
yes that is right